src/main/java/com/redhat/devtools/intellij/knative/kn/KnCli.java

```diff
@@ -308,6 +309,48 @@ public void deployFunc(String namespace, String path, String registry, String im
         return args.toArray(new String[0]);
     }
 
+    @Override
+    public void invokeFunc(InvokeModel model) throws IOException {
+        ExecHelper.executeWithTerminal(project, KNATIVE_TOOL_WINDOW_ID, getInvokeArgs("invoke", model));
```

Comment:
The `envVars` is missing so this is wrong on proxy enabled sites. I realized the whole KnCli is wrong when calling `executeWithTerminal` so maybe we should open a separate issue ?